core/system/src/core_armv8m/virq.c

```diff
@@ -117,7 +117,7 @@ static void virq_load_systick_ns(uint8_t box_id)
     if (g_virq_system_exception_state[box_id].systick.pending) {
         SCB_NS->ICSR |= SCB_ICSR_PENDSTSET_Msk;
```

Comment:
Might it be possible we set both `SCB_ICSR_PENDSTSET_Msk` and `SCB_ICSR_PENDSTCLR_Msk` with this write?

Reply:
Hi @Patater 

I think you are right. patch is updated and verified.
Thank you.